docs/source/user/vectorize.rst

```diff
@@ -297,6 +297,28 @@ demonstrate, we can  use an array of `float32` with the `init_values` function::
 In this case, there is no change to the `invals` array because the temporary
 casted array was mutated instead.
 
+.. note::
+   To ensure that ``@guvectorize`` updates `n`-th and `m`-th input arguments pass 
+   ``writable_args=(n, m)``.
+
```

Comment:
I think this note can be removed as it seems to serve only to provide a reference-like description of `writable_args`, which is introduced in a more tutorial-like fashion in the following paragraph.

To replace this, there should be a description in the reference manual for the `writable_args` kwarg.